crates/router/src/core/payments/tokenization.rs

```diff
@@ -1252,3 +1280,42 @@ pub fn update_connector_mandate_details(
 
     Ok(connector_mandate_details)
 }
+
+#[cfg(feature = "v1")]
+pub fn update_connector_mandate_details_status(
+    merchant_connector_id: Option<id_type::MerchantConnectorAccountId>,
+    mandate_details: Option<diesel_models::PaymentsMandateReference>,
```

Comment:
why this function takes both of these arguments as optional? the caller can decide this. Both arguments should be mandatory here